hack/testing/test-upgrade.sh

```diff
@@ -293,6 +298,9 @@ function verifyUpgrade() {
   local checkMigrate=${2:-false}
   local checkCDMMigrate=${3:-false}
 
+### check configmap patched for MIN_MASTERS
+  [[ -z "$(oc get configmap logging-elasticsearch | grep MIN_MASTERS)" ]] && return 1
```

Comment:
I think you want to use `-n` here 
The way this currently reads is "if the configmap does not have MIN_MASTERS, error"